src/components/Header.tsx

```diff
@@ -14,19 +14,22 @@ export const Header = () => {
   };
 
   return (
-    <div className="Header">
-      <div className="Inset">
-        <ProjectLogo title="Demo Wallet" />
-        {account.isAuthenticated && (
-          <TextButton onClick={() => setModalVisible(true)}>
-            Sign out
-          </TextButton>
-        )}
-      </div>
+    <>
+      <Layout.Header
+        projectTitle="Demo Wallet"
+        onSignOut={
+          account.isAuthenticated ? () => setModalVisible(true) : undefined
```

Comment:
pretty NIT: but I think it's best practice to use `null` here instead of `undefined` because `null` denotes that you are explicitly saying "do nothing"

Reply:
The idea here is to make `onSignOut` an optional prop, so I thought that `undefined` would be clear, but looks like it's not. 😅 Should we use conditional prop spread instead? I wouldn't want to add an extra type `null` to the types [here](https://github.com/stellar/stellar-design-system/blob/main/%40stellar/design-system/src/components/Layout/index.tsx#L39), as it should be just an optional prop with a function type.

Reply:
Would this work better?
```ts
onSignOut={() => account.isAuthenticated && setModalVisible(true)}
```

Reply:
Makes sense. Yeah, I think a conditional prop spread is the way to go 👍

Reply:
👍 Updated in [this PR](https://github.com/stellar/stellar-demo-wallet/pull/216) (I don't want to make changes to this one since it will be the base).